src/modules/vtol_att_control/standard.cpp

```diff
@@ -335,22 +290,21 @@ void Standard::update_transition_state()
 		q_sp.copyTo(_v_att_sp->q_d);
 		_v_att_sp->q_d_valid = true;
 
-		hrt_abstime btrans_start;
-		btrans_start = _vtol_schedule.transition_start + uint64_t(_params_standard.reverse_delay) * 1000000.0f;
+		float btrans_start = (float)_vtol_schedule.transition_start + _params_standard.reverse_delay * 1e6f;
+
 		_pusher_throttle = 0.0f;
 
-		if (hrt_absolute_time() >= btrans_start) {
+		if (time_now >= btrans_start) {
 			// Handle throttle reversal for active breaking
-			float thrscale = (float)hrt_elapsed_time(&btrans_start) / (_params_standard.front_trans_dur *
-					 1000000.0f);
+			float thrscale = (time_now - btrans_start) / (_params_standard.pusher_ramp_dt * 1e6f);
```

Comment:
@sanderux I need your input here. You were using the parameter for the front transition duration for this calculation related to active breaking. I introduced a parameter for the pusher ramp up time so that we don't need to abuse the front transition time param anymore. Can you plz check if I used the correct param?